ci/gpu/build.sh

```diff
@@ -55,6 +55,15 @@ conda install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
       "rapids-notebook-env=$MINOR_VERSION.*" \
       "rapids-doc-env=$MINOR_VERSION.*"
 
+
+export CUDA_PATH=/usr/local/cuda
+pip install cupy==8.0.0b5
```

Comment:
Forgot: are we sure we want to test with cupy beta only? I feel this might make it so that we could end up with bugs in the mainstream version

Reply:
Yeah, we'll want to be on cupy 7.x for this release, maybe this was previous debugging code?

Reply:
Ok fine, I skipped the tests that required the most recent version of CuPy. We'll be able to re-enable them for 0.16.